.env.example

```
@@ -1 +1 @@
-NEXTAUTH_SECRET=YOUR_TOKEN_HERE
\ No newline at end of file
+NEXTAUTH_SECRET=...
```

Es importante dejar claro en este .env.example las variables de entorno que debería tener el proyecto, 

NOTA:
Si más adelante necesitamos más variables de entorno, es fundamental que también se añadan a este archivo los ejemplos (no los valores reales)

```suggestion
NEXTAUTH_SECRET=<Your_token_here>
```